### DISCUSSION TITLE
Environment variable override for name

### DISCUSSION THREAD

**Discussion by ghost** [pi0/config-dir#4] [General]
There may be lots of opinions on a default name. It may be nice to offer an environment variable for the name, like the [XDG](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) does. I don't have much to suggest as far as naming, but if you wanted to be pretty general you could start with `ES_` for EcmaScript or `JS_` for JavaScript. While XDG specifies full path, this variable could just be used to specify the naming convention used for the single-level directory.

**Comment by pi0** [comment]
Thanks for the suggestion. It is a good point however for tooling, it might become tricky because language servers already have restricted hard-coded places to recognize tool configurations from (for example `tailwind.config.js`).

If we are to think about this possibility to specify with an env like `CONFIG_DIR`, we need to make sure it is set **system-wide** and also IDEs respect it (which is unlikely, even global XDG envs are often not respected...)
